alvr/server_core/src/statistics.rs

```diff
@@ -297,14 +300,13 @@ impl StatisticsManager {
         }
     }
 
-    pub fn video_pipeline_latency_average(&self) -> Duration {
-        self.total_pipeline_latency_average.get_average()
+    pub fn motion_to_photon_latency_average(&self) -> Duration {
+        self.motion_to_photon_latency_average.get_average()
     }
 
     pub fn tracker_pose_time_offset(&self) -> Duration {
         // This is the opposite of the client's StatisticsManager::tracker_prediction_offset().
         self.steamvr_pipeline_latency
-            .saturating_sub(self.total_pipeline_latency_average.get_average())
     }
```

Comment:
Well I cannot explain it either 😅 basically `total_pipeline_latency_average` usage was broken because there is no place actually pushing samples into it, so it was always 0. some time ago (maybe 1 year) i pushed a fix for this, and I got reports that controller tracking became wonkier, but I cannot explain why. I had my reasons for how I designed the controller prediction logic like this.